features/steps/common_http.py

```diff
@@ -187,3 +188,12 @@ def check_for_null_attribute(context, attribute):
 
     assert attribute in json, f"Attribute {attribute} is not returned by the service"
     assert json[attribute] is None, f"Attribute {attribute} should be null"
+
+
+@given(u'the service is running')
+@then(u'the service is running')
+def is_server_running(context):
```

Comment:
I'm not a fan of this step definition. So far what we've been doing is starting the service on each scenario with the corresponding arguments. Even if content-.service is a simpler service, I'd do it the same way.